README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+It's an easy way of describing to the browser what that specific content is because the browser doesn't look at the class and id labels to quickly pull information about the page. 
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+display: inline; are elements that reside inside of the display: block;
+
 3. What are the 4 areas of the box model?
 
+Margin, Border, Padding, Content
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+The cross axis
+
 5. Explain why git is valuable to a team of developers.
 
+It enables them to work together on projects without overwriting each other's work.
```

Comment:
Great job, as you progress through lambda you will use git and github daily and will become super familiar with all of its features aside from team collaboration.